src/Expander/Task.hs

```diff
@@ -36,18 +37,19 @@ data ExpanderTask
   | AwaitingDefn Var Ident Binding SplitCorePtr Ty SplitCorePtr Syntax
     -- ^ Waiting on var, binding, and definiens, destination, syntax to expand
   | AwaitingType SplitTypePtr [AfterTypeTask]
-  | ExpandDecl DeclPtr Scope Syntax DeclValidityPtr
-    -- ^ Where to put it, the scope, the decl, where to put the phase
-  | ExpandMoreDecls ModBodyPtr Scope Syntax DeclValidityPtr
-    -- Depends on the binding of the name(s) from the decl and the phase
```

Comment:
Now that I'm calling `expandDeclForm` instead of `forkExpandDeclForm`, I no longer need the `ExpandDeclForm` task.